doxygen/00_MainPage.dox

```diff
@@ -18,7 +18,7 @@ GeNN is a software package to enable neuronal network simulations on
 NVIDIA GPUs by code generation. Models are defined in a simple C-style
 API and the code for running them on either GPU or CPU hardware is
 generated by GeNN. GeNN can also be used through external
-interfaces. Currently there are interfaces for \ref SpineML and for [Brian](http://brian2.readthedocs.io) via \ref brian2genn "Brian2GeNN".
+interfaces. Currently there are interfaces for \ref SpineML, a \ref PyGeNN and a [Brian](http://brian2.readthedocs.io) interface via \ref brian2genn "Brian2GeNN".
```

Comment:
SpineML, \ref PyGeNN and [Brian]... (no "a"s)

Reply:
This was intentional as this renders to:
> Currently there are interfaces for SpineML and SpineCreator, a Python interface (PyGeNN) and a Brian interface via Brian2GeNN.

Reply:
hum ... "and SpineCreator" appearing magically ;-) ?
Ok, fine by me.